Tests/WorkspaceTests/InitTests.swift

```diff
@@ -414,15 +448,12 @@ final class InitTests: XCTestCase {
                 fileSystem: localFileSystem
             )
             try initPackage.writePackageStructure()
-            
-            await XCTAssertBuilds(
-                packageRoot,
-                buildSystem: .native,
-            )
+
+            await expectBuilds(packageRoot, buildSystem: .native)
```

Comment:
suggestion: can we also augment the suite to execute all the tests against the `swiftbuild` build system as well.